src/formio/components/fieldset.ts

```diff
@@ -0,0 +1,50 @@
+import {AnyComponentSchema, LayoutComponentSchema, OFExtensions} from '..';
+
+type TranslatableKeys = 'label';
+
+/**
+ * The fieldset component schema.
+ *
+ * Fieldsets are used to manage group fields together. They can improve accessibility
+ * by grouping related fields together.
+ *
+ * The `key` property has no effect on the submission data structure (it does not create
+ * a nesting level, the component is purely presentational).
+ *
+ * The label is displayed as the legend element, while the upstream component uses an
+ * explicit legend property for this purpose.
```

Comment:
I'm not sure I understand this sentence:
- > The label is displayed as the legend element

legend == [`<legend>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend)?
- > while the upstream component uses an explicit legend property for this purpose.

upstream = formio? What does _explicit_ means here?

Reply:
Ok nice, I'd be interested to see how this will be implemented on the form builder as I've used `<fieldset>` with `<legend>` as well.